lib/libxdp/xsk.c

```diff
@@ -398,6 +399,32 @@ static enum xdp_attach_mode xsk_convert_xdp_flags(__u32 xdp_flags)
 	return XDP_MODE_NATIVE;
 }
 
+#define MAX_DEV_QUEUE_PATH_LEN 64
+
+static int xsk_get_queues_from_sysfs(const char* ifname) {
+	DIR *dir;
```

Comment:
This is a comment for Toke I guess: are we using reverse Christmas tree in libxdp or not? If yes, then this is in the wrong order. If not, just leave it as is.

Reply:
Yeah, please try to keep reverse x-mas tree :)